invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -50,6 +55,34 @@ function formatDocumentToTableView(document, volume = null) {
   return serialized;
 }
 
+function formatMostLoanedDocumentToTableView(document, fromDate, toDate) {
+  let serialized = formatDocumentToTableView(document);
+  if (serialized.Title) {
+    const documentUrl = BackOfficeRoutes.documentDetailsFor(serialized.ID);
+    serialized['Title'] = <Link to={documentUrl}>{serialized.Title}</Link>;
+  }
+  if (!isEmpty(document.metadata)) {
+    if (document.metadata.loan_count) {
+      const query = loanApi
```

Comment:
I think that we should do queries in the `actions`, not in a formatter.
Probably the action method should do the 2 queries, or the backend also return the loans count.